src/main/kotlin/value/VMap.kt

```diff
@@ -12,6 +12,15 @@ data class VMap(val v: MutableMap<String, Value>): Value() {
     override fun toString() = "[${v.entries.joinToString()}]"
     override fun asString() = v.entries.joinToString(", ")
 
+    override fun plus(a2: Value) = when (a2) {
+        is VMap -> make(mutableMapOf<Value, Value>().apply {
+                realKeys.keys.forEach { set(realKeys[it]!!, v[it]!!) }
+                a2.realKeys.keys.forEach { set(a2.realKeys[it]!!, a2.v[it]!!)}
+            }
+        )
```

Comment:
I am probably missing something, but it seems like you could do all this with the kotlin map `+` operator?  ( `make(realKeys + a2.realkeys)` )

Reply:
OMG you're right.  I must have had a stroke when I wrote this.  Thanks for the golfshot ⛳ 😁